build-mingw-w64-tools.sh

```diff
@@ -83,7 +85,7 @@ cd mingw-w64-tools/widl
 [ -z "$CLEAN" ] || rm -rf build${CROSS_NAME}
 mkdir -p build${CROSS_NAME}
 cd build${CROSS_NAME}
-../configure --prefix="$PREFIX" --target=$ANY_ARCH-w64-mingw32 --with-widl-includedir="$INCLUDEDIR" $CONFIGFLAGS
+CC=$arch-w64-mingw32-clang ./configure --prefix="$PREFIX" --target=$ANY_ARCH-w64-mingw32 --with-widl-includedir="$INCLUDEDIR" $CONFIGFLAGS
```

Comment:
This particular change is explicitly wrong. `build-mingw-w64-tools.sh` produces build tool that run on the current host, those tools aren't supposed to be cross compiled. (And in this context, `$arch` is undefined.)

Reply:
Right, I'll fix that up.